src/components/ActionsMenu/Item/ActionsMenuItem.st.css

```diff
@@ -0,0 +1,97 @@
+:import {
+    -st-from: "../../../common/formatters.st";
+    -st-named: color, fallback, font, applyOpacity;
+}
+
+:import {
+    -st-from: "../../Text/Text.st.css";
+    -st-default: Text;
+}
+
+:vars {
+    /*Defaults*/
+    ItemFontDesktop: "{theme: 'Body-M', size: '16px', lineHeight: '24px'}";
+    ItemSubtitleFontDesktop: "{theme: 'Body-M', size: '14px', lineHeight: '20px'}";
+    ItemFontMobile: "{theme: 'Body-M', size: '14px', lineHeight: '24px'}";
+    ItemSubtitleFontMobile: "{theme: 'Body-M', size: '12px', lineHeight: '20px'}";
+}
+
+:vars {
+    /*Overrides*/
+    ItemFont:  --overridable;
+    ItemSubtitleFont:  --overridable;
+}
+
+.root {
+    -st-states: disabled, mobile;
+    height: 40px;
+    padding: 8px 24px;
+    display: flex;
+    cursor: pointer;
+    position: relative;
+}
+
+.root:hover {
+    background-color: applyOpacity(color(color-5), 0.06);
+}
+
+.root .container {
+    display: flex;
+    flex: 1;
+    position: absolute;
+}
+
+.root:disabled {
+    opacity: 0.3;
+    cursor: default;
+}
+
+.root:disabled:hover {
+    background-color: inherit;
+}
+
+.root .text {
+    -st-extends: Text;
+    -st-mixin: Text(
+        MainTextColor color(color-5)
+    );
+    display: flex;
+    flex-direction: column;
+    width: 100%;
```

Comment:
This should also have a `justify-content: center;` since we want the text to always be in the middle of the item